neqo-qpack/src/huffman.rs

```diff
@@ -4,9 +4,12 @@
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
 
+#![allow(clippy::pedantic)]
```

Comment:
Is there a particular lint that is awkward here?

Reply:
This implementation will change once #48 is done. So I decided not to spend too much time on this and do it as a part of the #48 .